README.md

```diff
@@ -1,5 +1,8 @@
-![Mastodon](https://i.imgur.com/NhZc40l.png)
-========
+<h1><picture>
+  <source media="(prefers-color-scheme: dark)" srcset="./lib/assets/wordmark.dark.png?raw=true">
+  <source media="(prefers-color-scheme: light)" srcset="./lib/assets/wordmark.light.png?raw=true">
+  <img alt="Mastodon" src="./lib/asssets/wordmark.light.png?raw=true" height="34">
+</picture></h1>
```

Comment:
```suggestion
<h1>
  <picture>
    <source media="(prefers-color-scheme: dark)" srcset="https://raw.githubusercontent.com/mastodon/mastodon/change-branding/lib/assets/wordmark.dark.png">
    <source media="(prefers-color-scheme: light)" srcset="https://raw.githubusercontent.com/mastodon/mastodon/change-branding/lib/assets/wordmark.light.png">
    <img alt="Mastodon" src="https://raw.githubusercontent.com/mastodon/mastodon/change-branding/lib/assets/wordmark.light.png" height="34">
  </picture>
</h1>
```

`! []()` without using Markdown syntax such as `[]()`, the conversion of file paths to URLs does not seem to occur when using the `img` element or the `source` element.

Reply:
But the images do show up when you view the file. Your suggestion would stop working as soon as the PR is merged and the branch stops existing.

Reply:
Looks like they are aware of this and it might get fixed: https://github.com/github/feedback/discussions/16910#discussioncomment-2787270

Reply:
> Your suggestion would stop working as soon as the PR is merged and the branch stops existing.

Yes, you must replace the branch name with main before merge into the main branch.

Reply:
They fixed it.